docs/apis.md

```diff
@@ -30,19 +30,100 @@ See the Prometheus documentation for [more information on the Prometheus remote
 
 Cortex supports the Prometheus' [alerts](https://prometheus.io/docs/prometheus/latest/querying/api/#alerts) and [rules](https://prometheus.io/docs/prometheus/latest/querying/api/#rules) api endpoints. This is supported in the Ruler service and can be enabled using the `experimental.ruler.enable-api` flag.
 
-`GET /api/prom/api/v1/rules` - List of alerting and recording rules that are currently loaded
+```
+GET /prometheus/api/v1/rules
+```
+
+List of alerting and recording rules that are currently loaded by the Ruler. This endpoint retrieves the running rule groups in each ruler for a user, merges them and returns the response to the caller.
```

Comment:
```suggestion
List of alerting and recording rules that are currently loaded in the Ruler. This endpoint retrieves the running rule groups in each ruler for a user, merges them and returns the response to the caller.
```

Reply:
I find the last sentence an implementation on how the ruler works when it is sharded - I don't think it's important for the API but rather for the ruler service description or the definition above.